search_events_app/models/event.py

```diff
@@ -1,5 +1,6 @@
 class Event:
-    def __init__(self, name, url, country=None, feature=[]):
+
+    def __init__(self, name, url, country=None, feature=[], **kwargs):
```

Comment:
never put a mutable as a default argument, should be `feature=None` and then:
`if not feature:
    feature = []`
https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html